open-company/open-company-web
# Comments

Part 1 of this card: https://trello.com/c/aBhDvqzr

Static comments. This is missing the delete and edit comments. For now it retrieves the comments for each rendered entry in the topic view. When the user clicks on the comments icon it toggles the comments UI.
The comments button is shown only if there is the `comments` rel in the entry data. The new comment component at the bottom of the comments container is shown only if the is the `comment` rel in the entry area.
So the comments icon won't be shown on a new entry (not yet saved).

To test:
- open a board and click on a topic (with more entries if possible)
- check the number of comments besides the comments icon (shown only if >0)
- click it
- check that the comments are shown
- add a comment
- check that the number of comments is increased in the comments button
- refresh the topic view and check that the number of comments is still good
- toggle the comments for more entries to make sure the right data are always shown
